about.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="mobile.css" />
+    <link rel="stylesheet" href="desktop.css" />
+    <link rel="stylesheet" href="about-mobile.css" />
+    <link rel="stylesheet" href="about-desktop.css" />
+    <title>About WC Global competition</title>
+  </head>
+  <body>
+     
+    <!-- HEADER -->
+    <header>
+      <!-- MOBILE MENU -->
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+      <div class="desktop-nav-bar">
+        <div class="logo-div">
+          <a href="index.html"
+            ><img
+              class="logo-img"
+              src="./images/icons/WC-logo-icon.png"
+              alt="World chef logo"
+          /></a>
+          <p class="logo-title">
+            WC Global <br />
+            Competition
+          </p>
+        </div>
+        <!-- NAVBAR -->
+        <nav class="navbar">
+          <ul class="nav-menu">
+            <li class="nav-item"><a href="about.html">About</a></li>
```

Comment:
- Your Project is awesome @Rachelwebdev, Great Job! 👍🏽 
- I'd like to kindly sugggest that it will be better if you replaced `li class="nav-item"><a href="about.html">About</a></li>` to `li class="nav-item"><a href="index.html">Home</a></li>` inorder to navigate back to the home page from the about page. This will enable good user experience. Can we fix that?


Here is what I mean👇🏽 
| Your mobile menu on about.html  |
| :---:   | 
|<img width="395" alt="Screenshot 2023-01-03 at 11 19 45" src="https://user-images.githubusercontent.com/99246359/210338892-59e221f8-10f4-4654-9180-952db772743f.png">
  |

Reply:
Oops. It didn't display like this on my screen. However, I would take a look into why this bug is there. Thanks for the review.